test/e2e/e2e.go

```diff
@@ -584,6 +606,80 @@ var _ = ginkgo.Describe("e2e control plane", func() {
 		framework.ExpectNoError(err, "one or more nodes failed to go back ready, schedulable, and untainted")
 	})
 
+	ginkgo.FIt("should provide Internet connection continuously when all pods are killed on node running master instance of ovnkube-master.", func() {
```

Comment:
```suggestion
	ginkgo.It("should provide Internet connection continuously when all pods are killed on node running master instance of ovnkube-master.", func() {
```
Didnt review the rest of this but just noticed this.